### PR TITLE
FAQ ENA.ExposureNotificationError-Error2

### PR DESCRIPTION
**Pull request by dsarkar**
closes https://github.com/corona-warn-app/cwa-website/issues/743

### CODE REVIEW

**Comment by dsarkar** [comment]
EN
image

DE
image

File: src/data/faq.json [hunk 1]
```diff
@@ -65,11 +65,12 @@
...
+                        "title": "[Apple/iOS] Can’t enable exposure logging / Empty screen on iOS 13.7/14 / ENA.ExposureNotificationError-Error2",
                         "anchor": "ios_black_screen",
                         "active": true,
                         "textblock": [
-                            "There is a known issue under iOS 13.7 / 14 where in rare cases the Exposure Notifications can not be activated again after an update. The result is a black screen in the iOS settings.",
+                            "There is a known issue under iOS 13.7 / 14 where in rare cases the Exposure Notifications cannot be activated again after an update. The result is a black screen in the iOS settings.",
+                            "Also, rare cases (after updates typically) with the error notification 'ENA.ExposureNotificationError-Error2' are known.",
                             "To fix the problem, proceed as follows:",
                             "Settings > General > Reset > Reset Location and Privacy",
                             "Afterwards the permissions in the Corona-Warn App have to be assigned again. You will be prompted to do so when the App is started."
```
Comment (MikeMcC399) [inline src/data/faq.json:76]: Corona-Warn App => Corona-Warn-App

File: src/data/faq_de.json [hunk 2]
```diff
@@ -65,11 +65,12 @@
...
-                        "title": "[Apple/iOS] Leerer Bildschirm auf iOS 13.7 / 14",
+                        "title": "[Apple/iOS] Leerer Bildschirm auf iOS 13.7/14 / ENA.ExposureNotificationError-Fehler2",
                         "anchor": "ios_black_screen",
                         "active": true,
                         "textblock": [
                             "Es gibt ein bekanntes Problem unter iOS 13.7 / 14 wo es in seltenen Fällen vorkommen kann dass sich die Begegnungsmitteilungen nach einem Update nicht mehr aktivieren lassen. Das Resultat ist ein schwarzer Bildschirm in den iOS Einstellungen.",
+                            "Es gibt auch seltene Fälle (typischerweise nach Updates) mit der Meldung 'ENA.ExposureNotificationError-Fehler2'. ",
                             "Um das Problem zu beheben gehen Sie wie folgt vor:",
                             "Einstellungen > Allgemein > Zurücksetzen > Standort und Datenschutz",
                             "Im Anschluss müssen die Berechtigungen in der Corona-Warn App erneut vergeben werden. Dazu werden Sie beim Start der App aufgefordert."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:76]: Corona-Warn App => Corona-Warn-App

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I suggest changing Corona-Warn App to Corona-Warn-App (add hyphen) to be consistent with how it is written in other FAQ entries.

File: src/data/faq_de.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq_de.json:76]: ok thanks

**Review by heinezen — APPROVED** [review]

**Comment by heinezen** [comment]
LGTM :)

**Pull request merged**